### ISSUE TITLE
Floating Ghost Piece

### ISSUE THREAD

**Issue by XeroOl** [OpenRCT2/OpenRCT2#4924]
**OS:** Windows 8
**Version:**0.0.5.0 
**Commit/Build:** ac32e7d

Every once in a while, when I'm building a coaster, a floating ghost piece will remain on the screen even if I remove the tracks leading up to it. I believe it happens when you delete pieces really quickly on a laggy multiplayer server.

- [ ] Reproducible in RCT2 (vanilla)? I doubt it
- [x] Multiplayer? I think this is multiplayer only, as I have only been able to reproduce it on multiplayer.

**Steps to reproduce:**
1. Join a server
2. Build a roller coaster
3. Quickly delete multiple segments

**Dump file**

**Screenshots / Video:**
http://imgur.com/a/cHlq6

**Save game:**

**Comment by devnoname120** [comment]
Might be related: https://github.com/OpenRCT2/OpenRCT2/issues/259

**Comment by Broxzier** [comment]
I don't think this is related to #259, as that one was fixed with https://github.com/OpenRCT2/OpenRCT2/commit/4811d8ee8385c528e14b73a355a6b58d7d03955f.

**Comment by devnoname120** [comment]
@Broxzier I just tried and can confirm that #259 isn't fixed in b1018ae.

**Comment by Broxzier** [comment]
It's crash for me with that method now, instead of the ghost pieces staying.

**Comment by 733737** [comment]
fixed?
